.github/workflows/build-magma-windows.yml

```diff
@@ -32,7 +32,7 @@ jobs:
       - name: Build MAGMA and push to S3
         run: windows/internal/build_magma.bat
       - name: Save as artifact
-        uses: actions/upload-artifact@v2
+        uses: actions/upload-artifact@v4
```

Comment:
This change is good, but do you mind move it to the separate PR?

Reply:
Thanks! It appears that this maybe needed, as test is failing with "This request has been automatically failed because it uses a deprecated version of `actions/upload-artifact: v2`" if unchanged